src/ui/services/user.ts

```diff
@@ -41,15 +41,14 @@ const getUsers = async (
   setData: SetStateCallback<UserData[]>,
   setAuth: SetStateCallback<boolean>,
   setErrorMessage: SetStateCallback<string>,
-  query: Record<string, string> = {},
 ): Promise<void> => {
-  const url = new URL(`${API_BASE}/api/v1/user`);
-  url.search = new URLSearchParams(query).toString();
-
   setIsLoading(true);
 
   try {
-    const response: AxiosResponse<UserData[]> = await axios(url.toString(), getAxiosConfig());
+    const response: AxiosResponse<UserData[]> = await axios(
+      `${API_BASE}/api/v1/user`,
+      getAxiosConfig(),
```

Comment:
I'm worried that if API_BASE is relative here axios still won't be able to resolve it because we're not setting baseUrl in the axiosConfig. which is imported from auth.js:

https://github.com/andypols/git-proxy/blob/455513a64239d9ca0533a1963e66a7b4a3322ec1/src/ui/services/auth.js#L30-L39

In the other services that are still JS they work out baseUrl for themselves and fallback to `location.origin`:

https://github.com/andypols/git-proxy/blob/455513a64239d9ca0533a1963e66a7b4a3322ec1/src/ui/services/repo.js#L4-L6

whjch apiBase.ts does not do...